src/render/vulkan/VULKAN_PixelShader_Common.incl

```diff
@@ -120,45 +117,8 @@ float4 GetInputColor(PixelShaderInput input)
 
     if (texture_type == TEXTURETYPE_NONE) {
         rgba = 1.0;
-    } else if (texture_type == TEXTURETYPE_RGB) {
-        rgba = texture0.Sample(sampler0, input.tex);
-    } else if (texture_type == TEXTURETYPE_NV12) {
-        float3 yuv;
-        yuv.x = texture0.Sample(sampler0, input.tex).r;
-        yuv.yz = texture1.Sample(sampler0, input.tex).rg;
-
-        yuv += Yoffset.xyz;
-        rgba.r = dot(yuv, Rcoeff.xyz);
-        rgba.g = dot(yuv, Gcoeff.xyz);
-        rgba.b = dot(yuv, Bcoeff.xyz);
-        rgba.a = 1.0;
-    } else if (texture_type == TEXTURETYPE_NV21) {
-        float3 yuv;
-        yuv.x = texture0.Sample(sampler0, input.tex).r;
-        yuv.yz = texture1.Sample(sampler0, input.tex).gr;
-
-        yuv += Yoffset.xyz;
-        rgba.r = dot(yuv, Rcoeff.xyz);
-        rgba.g = dot(yuv, Gcoeff.xyz);
-        rgba.b = dot(yuv, Bcoeff.xyz);
-        rgba.a = 1.0;
-    } else if (texture_type == TEXTURETYPE_YUV) {
-        float3 yuv;
-        yuv.x = texture0.Sample(sampler0, input.tex).r;
-        yuv.y = texture1.Sample(sampler0, input.tex).r;
-        yuv.z = texture2.Sample(sampler0, input.tex).r;
-
-        yuv += Yoffset.xyz;
-        rgba.r = dot(yuv, Rcoeff.xyz);
-        rgba.g = dot(yuv, Gcoeff.xyz);
-        rgba.b = dot(yuv, Bcoeff.xyz);
-        rgba.a = 1.0;
     } else {
-        // Error!
-        rgba.r = 1.0;
-        rgba.g = 0.0;
-        rgba.b = 0.0;
-        rgba.a = 1.0;
+        rgba = texture0.Sample(sampler0, input.tex).rgba;
```

Comment:
The texture type constants can be removed from here and SDL_render_vulkan.c, right?

Reply:
Fixed